Doc/library/urllib.request.rst

```diff
@@ -42,6 +42,9 @@ The :mod:`urllib.request` module defines the following functions:
    the global default timeout setting will be used).  This actually
    only works for HTTP, HTTPS and FTP connections.
 
+   Warning: the global default timeout value is set with no timeout, which may
```

Comment:
Perhaps instead a `.. note::` (see L720 below) can be used here instead to stand out a bit more instead of the 'Warning:'.

Reply:
Sounds good, moved into a note.

Reply:
Also one more instance right below that could also be moved to a note.